modules/geolocation/geolocation.py

```diff
@@ -670,3 +689,12 @@ def map_location_from_pixel(self, transformationMatrix, pixels):
         self.__logger.debug("geolocation/map_location_from_pixel: Returned " + str(geoCoordinates))
 
         return geoCoordinates
+
+    def write_locations(self, locations):
+        # write the txt to save inside mapLabelling folder
+        # https://stackoverflow.com/questions/8024248/telling-python-to-save-a-txt-file-to-a-certain-directory-on-windows-and-mac
+        save_path = 'C:/Users/Owner/Documents/Jeffrey/School/Waterloo/Co-op/WARG/computer-vision-python/modules/mapLabelling'
+        completeName = os.path.join(save_path, 'new.txt')
+        with open(completeName, 'a') as f:
+            f.write('\n'.join([','.join(['{:4}'.format(item) for item in row]) for row in locations]))
+            f.write('\n')
```

Comment:
Have the function return `True` on success, `False` on failure (there may not be a failure in which case return true at the end).

Add an empty line to the end of geolocation.py .